RecentRevisions.php

```diff
@@ -108,11 +108,36 @@ function RecentRevisions() {
 
 
 function RecentRevisions_Init() {
-	wp_add_dashboard_widget( 'RecentRevisions', __( 'Recent Revisions', 'recent-revisions' ), 'RecentRevisions', 'RecentRevisions_Setup');
+	$options = RecentRevisions_Options();
+
+	switch ($options['permissions']) {
+		case 'subscriber':
+			$capability = 'read';
+			break;
+		case 'contributor':
+			$capability = 'edit_posts';
+			break;
+		case 'author':
+			$capability = 'edit_published_posts';
+			break;
+		case 'editor':
+			$capability = 'edit_others_posts';
+			break;
+		case 'administrator':
+			$capability = 'manage_options';
+			break;
+		default:
+			$capability = 'edit_posts';

```

Comment:
Can you clarify, what the default role is, why is it the same permissions as contributor. I'm thinking that a more conservative default would be `read`?